src/go/rpk/pkg/cli/transform/list_test.go

```diff
@@ -106,9 +108,9 @@ func TestPrintSummaryView(t *testing.T) {
 	s := summarizedView(setupTestData())
 	cases := []testCase{
 		Text(`
-NAME      INPUT TOPIC  OUTPUT TOPIC     RUNNING  LAG
-foo2bar   foo          bar              2 / 3    7
-scrubber  pii          cleaned, munged  0 / 1    99
+NAME      INPUT TOPIC  OUTPUT TOPIC     RUNNING  LAG   COMPRESSION
```

Comment:
Is this useful? We don't print env vars and this feels like that configuration. Maybe in the detail view? Doesn't feel like there is a user story where someone wants this

Reply:
good point, yeah I suppose not.